NEWS.md

```diff
@@ -1,40 +1,52 @@
 # polars (development version)
+
+## BREAKING CHANGES
+
+- Replace the argument `reverse` by `descending` in all sorting functions. This
+  is for consistency with the upstream Polars (#291, #293).
+
 ## What's changed
 
-- New function `rename()` for `LazyFrame` and `DataFrame` (#239)
+- New method `$rename()` for `LazyFrame` and `DataFrame` (#239)
 - `<DataFrame>$unique()` and `<LazyFrame>$unique()` gain a `maintain_order` argument (#238).
-- New `pl$LazyFrame()` to quickly create a `LazyFrame`, mostly in examples or 
+- New `pl$LazyFrame()` to quickly create a `LazyFrame`, mostly in examples or
 for demonstration purposes (#240).
 - Polars is internally moving away from string errors to a new error-type called `RPolarsErr` both on rust- and R-side. Final error messages should look very similar (#233).
-- LazyFrame_columns, _schema, _dtypes implemented. Improvements to internal `RPolarsErr`. Also `RPolarsErr` will now print each context of the error on a separate line (#250).
+- `$columns()`, `$schema()`, `$dtypes()` for `LazyFrame` implemented (#250).
+- Improvements to internal `RPolarsErr`. Also `RPolarsErr` will now print each context of the error on a separate line (#250).
 - Fix memory leak on error bug. Fix printing of `%` bug. Prepare for renaming of polars classes (#252).
 - Add helpful reference landing page at `polars.github.io/reference_home` (#223, #264).
-- rust-polars' `simd` feature is now disabled by default. To enable it, set the environment variable
-  `RPOLARS_ALL_FEATURES` to `true` when build r-polars (#262).
+- Supports Rust 1.65 (#262, #280)
+  - rust-polars' `simd` feature is now disabled by default. To enable it, set the environment variable
```

Comment:
```suggestion
- rust-polars' `simd` feature is now disabled by default. To enable it, set the environment variable
```

Reply:
Thanks for taking a look at this.

The reason we disabled `simd` by default was to support Rust 1.65, so I think it makes sense to make it a child item.
What do you think?

Reply:
ah ok, but since it's the only one I think it makes sense to put it in the same sentence, like 
```
- Supports Rust 1.65. This means that rust-polars' `simd` feature is now disabled by default. To enable it, set the environment variable... (#262, #280)
```

Reply:
That makes sense.

But the addition of the new profile `release-optimized` is also related to this, so I will add a new child item.